src/map-interactions.react.js

```diff
@@ -138,7 +145,7 @@ export default class MapInteractions extends Component {
   @autobind
   _onMouseDrag(event) {
     const pos = this._getMousePos(event);
-    this.setState({pos});
+    this.setState({pos, dragged: true});
```

Comment:
Since this is a flag to indicate if the mouse moved while being "down", how about `didDrag`?